src/datePickerAndroid.rei

```diff
@@ -0,0 +1,24 @@
+[@bs.scope "DatePickerAndroid"] [@bs.module "react-native"] external dateSet : string =
```

Comment:
`externals` are exposed in `rei`s as let bindings. In this case it'd be `let dateSet : string;`

Reply:
thanks :)